libcbtest/callback.c

```diff
@@ -0,0 +1,9 @@
+#include <string.h>
+
+typedef int (*callback)(const char *, int);
+
+int printString(const void *fp, const char *s) {
```

Comment:
What about just `callCallback`? This function itself doesn't call printf.

Reply:
Done